README.md

```diff
@@ -6,7 +6,7 @@ Relay is a JavaScript framework for building data-driven React applications.
 * **Colocation:** Queries live next to the views that rely on them, so you can easily reason about your app. Relay aggregates queries into efficient network requests to fetch only what you need.
 * **Mutations:** Relay lets you mutate data on the client and server using GraphQL mutations, and offers automatic data consistency, optimistic updates, and error handling.
 
-[Learn how to use Relay in your own project.](https://facebook.github.io/relay/docs/getting-started.html)
+[Learn how to use Relay in your own project](https://facebook.github.io/relay/docs/getting-started.html) or read a [comprehensive introduction](https://learnrelay.org).
```

Comment:
Let's add a note clarifying that Learn Relay is a community-driven project and not an official source of documentation. How about:

`[Learn how to use Relay in your own project](https://facebook.github.io/relay/docs/getting-started.html) or check out the community-driven introduction at [Learn Relay](https://learnrelay.org).`